ScalaLoader/src/main/java/xyz/janboerman/scalaloader/plugin/description/ScalaVersion.java

```diff
@@ -19,13 +19,9 @@
             "https://bintray.com/bintray/jcenter/download_file?file_path=org%2Fscala-lang%2Fscala-reflect%2F2.12.8%2Fscala-reflect-2.12.8.jar",
             "https://bintray.com/bintray/jcenter/download_file?file_path=org%2Fscala-lang%2Fscala-library%2F2.12.8%2Fscala-library-2.12.8.jar"
     ),
-    v2_13_0_M4("2.13.0-M4", false,

```

Comment:
I'd prefer marking this with @Deprecrated(forRemoval = true) because people might be relying on this (at least I do for a personal project) and they're actually some api changes in the Scala standard library between these milestones. Yes I know that'll bump the java requirement to JDK9+ but Java 11 is out for almost a year now so I'm not worriedOnce I get to release version 1.0 of ScalaLoader we can remove the deprecated constants.

Reply:
If I want to set **forRemoval** I must set **release** to 9 or higher inside maven-compiler-plugin configuration. I don't know if it will break compatibility to be able to run ScalaLoader with JRE 8 so I prefer to just set as Deprecated and wait your answer :)

Reply:
It will break compatibility with JRE8 because the classfile version will be increased. JRE8 will then throw an unsupported major.minor error when it tries to load the classes. Marking it with the regular @Deprecated is fine with me, it'll still be compatible then. I'll merge once you commited those changes.